KNOWN_ISSUES.md

```diff
@@ -64,8 +78,9 @@ for a full list of known issues.
 
 ### Automatic Differentiation
 
-* [ ] Differentiation does not yet support functions with control flow.
+* [ ] 🚧 **Differentiation does not yet support functions with control flow.**
       ([TF-356](https://bugs.swift.org/browse/TF-356))
+* [ ] 🚧 **Higher-order differentiation is not yet supported.**
```

Comment:
cc @rxwei
I don't believe higher-order differentiation has a JIRA issue yet.
There are a bunch of related issues (e.g. [bundle representation](https://bugs.swift.org/projects/TF/issues/TF-383), [bidirectional differentiation](https://bugs.swift.org/browse/TF-382)). Do you feel any are worth mentioning here?

Reply:
I created [TF-459](https://bugs.swift.org/browse/TF-459). Feel free to mention it.

Reply:
Thanks! Mentioned in https://github.com/tensorflow/swift/pull/178/commits/153920800c29ca393d79acd2acab25d207a938a2.